s3tests_boto3/functional/test_s3.py

```diff
@@ -720,6 +1141,14 @@ def test_bucket_list_marker_none():
     response = client.list_objects(Bucket=bucket_name)
     eq(response['Marker'], '')
 
+def test_bucket_list_continuationtoken_none():
```

Comment:
Can we get `listv2` instead of `list`here, between `bucket` and `continuationtoken`.

Reply:
Yup